### PR TITLE
GitHub CLI (#833)

### PR DESCRIPTION
**Pull request by Akuhana**
Adding mirrors for the GitHub CLI Commandlet

### CODE REVIEW

File: gh/os-mappings [hunk 1]
```diff
@@ -0,0 +1,3 @@
+windows=windows
+mac=macOS
+*=Linux
```
Comment (tobka777) [inline gh/os-mappings]: *Linux* should be lowercase (*linux*) as I see in releases.

**Review by tobka777 — CHANGES_REQUESTED** [review]
Thank you. It looks good. Only a small thing.

File: gh/os-mappings (hunk 1 above)
Comment (Akuhana) [reply gh/os-mappings]: Thank you for pointing that out! Not sure how I wrote linux in uppercase...

**Review by tobka777 — APPROVED** [review]
Perfect!

**Pull request merged**